openssl/src/pkey.rs

```diff
@@ -627,6 +626,36 @@ impl PKey<Private> {
             .map(|p| PKey::from_ptr(p))
         }
     }
+
+    /// Serializes a private key into a DER-formatted PKCS#8, using the supplied password to
+    /// encrypt the key.
+    ///
+    /// # Panics
+    ///
+    /// Panics if `passphrase` contains an embedded null.
+    pub fn private_key_to_pkcs8_passphrase(
+        &self,
+        cipher: Cipher,
+        passphrase: &[u8],
+    ) -> Result<Vec<u8>, ErrorStack> {
+        unsafe {
+            ffi::init();
```

Comment:
The PKey existing implies this has already happened so you can skip it.

Reply:
Ok, i removed it and rebased the branch to resolve the new conflicts.